src/plugins/intel_npu/src/backend/src/zero_executor.cpp

```diff
@@ -75,6 +75,11 @@ ZeroExecutor::ZeroExecutor(const std::shared_ptr<const ZeroInitStructsHolder>& i
                          _networkDesc->compiledNetwork.size(),
                          _networkDesc->compiledNetwork.data(),
                          nullptr};
+
+    // GET THE graph handle from the compiler
+    ze_graph_handle_t* originalPtr = static_cast<ze_graph_handle_t*>(networkDescription->graphHandleVoidPtr);
+    _graph = *originalPtr;
+
     zeroUtils::throwOnFail(
```

Comment:
Still create new handle instead of the handle passed from compiler

Reply:
Fixed in latest commit, skip this part (pfnCreate step) when the _graph return from compiler is not nullptr